_scripts/webpack.renderer.config.js

```diff
@@ -35,9 +35,12 @@ const config = {
   module: {
     rules: [
       {
-        test: /\.js$/,
-        use: 'babel-loader',
+        test: /\[tj]s$/,
+        loader: 'esbuild-loader',
         exclude: /node_modules/,
+        options: {
+          target: 'esnext'
```

Comment:
Again same as above we are still using an electron version that uses chromium 106